# dont forget to update this number according to following contents
number_of_detectors: 2

#------------------------------------------------------------------------------------------

sensor_0: # the names are fixed, just the number changes

  # by this name the device is found, run "roslaunch rospix list_devices" to find it
  name: 'Medipix2 ver 1.32 sn: 1096'

  # location of the equalization file
  # location is relative to a directory specified in launch file
  equalization: 'lite_rigaku.bpc' # *.bpc is needed

  # print percentage of sensor exposure time to real time
  print_utilization: true
  
  defaults:
    threshold: 340    # [-]
    bias: 10.0        # [Volt]
    exposure: 1.0     # [second]
    mode: 1           # [0 = MPX, 1 = TOT]
    clock: 20         # {10, 20, 40} Mhz are the only allowed values

    # dacs can be found in calibration protocol (or equalization protocol)
    dacs: [1,100,255,127,127,0,340,7,130,128,80,85,128,128]

#------------------------------------------------------------------------------------------

sensor_1: # the names are fixed, just the number changes

  name: 'dummy'
  
  photon_flux: 100            # [photons/s], whole number
  simulate_background: true   # should we simulation radiation background?
  n_images: 20                # how many images of radiation background do we have?

  defaults:
    exposure: 1.0             # [seconds]
